crates/notion-core/src/path/windows.rs

```diff
@@ -128,69 +91,22 @@ pub fn node_image_3p_bin_dir(_node: &str, _npm: &str) -> Fallible<PathBuf> {
     unimplemented!("global 3rd party executables not yet implemented for Windows")
 }
 
-pub fn yarn_image_root_dir() -> Fallible<PathBuf> {
-    Ok(image_dir()?.join("yarn"))
-}
-
-pub fn yarn_image_dir(version: &str) -> Fallible<PathBuf> {
-    Ok(yarn_image_root_dir()?.join(version))
-}
-
-pub fn yarn_image_bin_dir(version: &str) -> Fallible<PathBuf> {
-    Ok(yarn_image_dir(version)?.join("bin"))
-}
-
 pub fn launchbin_file() -> Fallible<PathBuf> {
-    Ok(local_data_root()?.join("launchbin.exe"))
+    Ok(notion_home()?.join("launchbin.exe"))
```

Comment:
As part of this refactoring you could also pull these executable paths up to `notion-core/src/path/mod.rs` and use `std::env::consts::EXE_SUFFIX` like in the layout code below, to eliminate some duplication. But, since the `path` module will be going away I that's not necessary.